### PR TITLE
Support `custom` variable type and empty values for conditionalTest

### PR DESCRIPTION
**Pull request by simPod**
_includes https://github.com/Vertamedia/clickhouse-grafana/pull/177_

### CODE REVIEW

**Comment by alex-zaitsev** [comment]
Hi @simPod , could you close preceeding PRs (#177 and #176) if this one includes everything, and also rebase from master. Thanks

**Comment by simPod** [comment]
@alex-zaitsev can we merge asap so I don't have to rebuild dist?

File: src/sql_query.ts [hunk 1]
```diff
@@ -558,57 +558,61 @@ export default class SqlQuery {
...
+
+                    const value = varG.current.value;
+
+                    if (
+                        // for query variable when all is selected
+                        // may be add another test on the all activation may be wise.
+                        (varG.type === 'query' && ((value.length == 1 && value[0] === '$__all')
+                            || (typeof value === 'string' && value === '$__all'))) ||
+                        // for textbox variable when nothing is entered
+                        (['textbox', 'custom'].includes(varG.type) && ['', undefined, null].includes(value))) {
```
Comment (alex-zaitsev) [inline src/sql_query.ts:593]: So this is the only significant change, everything else is just formatting

Comment (simPod) [reply src/sql_query.ts:593]: formatting is https://github.com/Vertamedia/clickhouse-grafana/pull/176

**Pull request merged**